server/datastore/mysql/apple_mdm.go

```diff
@@ -2269,11 +2284,29 @@ func (ds *Datastore) UpdateOrDeleteHostMDMAppleProfile(ctx context.Context, prof
 		detail = fmt.Sprintf("Failed to remove: %s", detail)
 	}
 
+	// Check whether we want to set a install operation as 'verifying' for an iOS/iPadOS device.
+	var isIOSIPadOSInstallVerifiying bool
+	if profile.OperationType == fleet.MDMOperationTypeInstall && profile.Status != nil && *profile.Status == fleet.MDMDeliveryVerifying {
+		if err := ds.writer(ctx).GetContext(ctx, &isIOSIPadOSInstallVerifiying, `
```

Comment:
Why does this need to use `ds.writer` instead of `ds.reader`?

Reply:
I'm playing safe for now in case of replication lag. (In case the host entry is created and then immediately profiles  are applied.)